spark/v3.1/spark/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -90,6 +90,8 @@
  *   <li><code>catalog-impl</code> - a custom {@link Catalog} implementation to use
  *   <li><code>default-namespace</code> - a namespace to use as the default
  *   <li><code>cache-enabled</code> - whether to enable catalog cache
+ *   <li><code>cache.case-sensitive</code> - whether the catalog cache should use case-sensitive
```

Comment:
Key may be a bit abstract?

"whether the catalog cache should compare table names in a case sensitive manner"?

Reply:
It is more than table names. Database name can also be an issue. Basically all the components of the TableIdentifier.
How about "whether the catalog cache should compare table identifiers in a case sensitive way"?

Reply:
Yea sounds good to me.